geode-core/src/main/java/org/apache/geode/internal/cache/DistributedRegion.java

```diff
@@ -2605,6 +2604,7 @@ protected void distributedRegionCleanup(RegionEventImpl event) {
       }
     }
     waitForCurrentOperations();
+    distAdvisor.close();
```

Comment:
It seems that this change is leading to the `ConnectionCloseSSLTLSDUnitTest.connectionWithHungReaderIsCloseableAndUnhangsReader()` test hanging

Reply:
Yeah, looks like this is the reason, I am going to leave GEODE-10287 out of this PR.